common/data_refinery_common/file_management.py

```diff
@@ -35,7 +35,7 @@ def _get_downloaded_name(batch: Batch) -> str:
     return os.path.basename(path)
 
 
-def _get_processed_name(batch: Batch) -> str:
+def get_processed_name(batch: Batch) -> str:
```

Comment:
Yes, it was previously only used within the `file_management` namespace but now it is used elsewhere as well.